src/core/os_android.c

```diff
@@ -334,6 +334,10 @@ void os_on_mouse_move(fn_mouse_move* callback) {
   //
 }
 
+void os_on_mousewheel_move(fn_mousewheel_move* callback) {
+  //
```

Comment:
Note for future: It would be possible to add checks for `AMOTION_EVENT_ACTION_SCROLL` and `AMOTION_EVENT_AXIS_HSCROLL`/`AMOTION_EVENT_AXIS_VSCROLL` to support this on Android.  I don't have any bluetooth mice to test w/ though.

Reply:
I'm interested in extending the supporting to Android.

BTW what's your opinion in funneling the phone touch events to mouse events like Love2D does?

Reply:
I think it would be totally fine to add touch emulation for the mouse inputs, just as a secret/unofficial "nice to have" feature.

Mouse can still be useful for VR (virtual desktop scenario), but I can't think of a VR use case for touch.  Even if you were using a phone as an input device for a VR experience, that would probably better be left as a separate phone app that communicates with the VR app.  So LÖVR probably isn't ready to add a full touch API.